Jenkinsfile

```diff
@@ -535,6 +547,9 @@ pipeline {
                 }
               }
             }
+            cleanup {
```

Comment:
See cleanup procedure in a previous version of CI

Reply:
@bakhtin Okay, I've added temp folder cleanup